functions/vpn.bash

```diff
@@ -169,3 +169,61 @@ setup_wireguard() {
   echo -n "$(timestamp) [openHABian] Generating QR to load config on the client side (download Wireguard app from PlayStore or AppStore)... "
   qrencode -t ansiutf8 </etc/wireguard/wg0-client.conf
 }
+
+
+## Install Tailscale from their own repo
+## Valid arguments: "install" or "remove"
+##
+##   install_tailscale(String action)
+##
+install_tailscale() {
+  local queryText="We will install the tailscale VPN client on your system. Use it to securely interconnect multiple openHAB(ian) instances.\\nSee https://tailscale.com/blog/how-tailscale-works/ for a comprehensive explanation how it creates a secure VPN. For personal use, you can get a free solo service from tailscale.com."
+  local serviceTargetDir="/lib/systemd/system"
+
+  if [[ -n "$INTERACTIVE" ]]; then
+    if (whiptail --title "VPN setup" --yes-button "Continue" --no-button "Cancel" --yesno "$queryText" 12 80); then echo "OK"; else echo "CANCELED"; return 1; fi
+  fi
+
+  if [[ "$1" == "remove" ]]; then
+    cond_redirect systemctl disable tailscaled.service
+    rm -f ${serviceTargetDir}/tailscale*
+    if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
+    if ! apt-get purge --yes tailscale; then echo "FAILED (purge tailscale)"; return 1; fi
```

Comment:
Remove the admin ssh key as well

Reply:
hmm, really? see https://github.com/openhab/openhabian/pull/1174#discussion_r507951840